util/rabbitmq/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { Event } from 'util/interfaces/events';
+import { validatorEvent } from 'util/validators';
+import { isEmpty, isNotEmpty } from 'util/api/util';
+import amqp from 'amqplib/callback_api';
+
+export const eventHandler = (event: Event) => {
+  const error = validatorEvent(event);
+  if (error) return error;
+  if (isEmpty(process.env.RABBITMQ_URL))
+    return { code: 500, message: 'RabbitMQ is not configured' };
+  if (isNotEmpty(process.env.NEXT_PUBLIC_E2E_TEST)) return;
+
+  amqp.connect(process.env.RABBITMQ_URL, (err: any, conn: any) => {
+    if (err) return { code: 500, message: 'RabbitMQ connection error' };
+    conn.createChannel((err: any, ch: any) => {
+      if (err) return { code: 500, message: 'RabbitMQ create channel error' };
+      const ex = 'events';
+      ch.publish(ex, 'private.integration', Buffer.from(JSON.stringify(event)));
+    });
+    conn.close();
```

Comment:
Ich bin mir hier nicht sicher, aber es könnte sein, dass die Verbindung geschlossen wird, bevor das Event verschickt wurde. Es wäre wahrscheinlich sicherer, wenn das `conn.close()` in ein kleines Timeout gepackt wird, so wie es in dem RabbitMQ Tutorial auch gemacht wird

Reply:
funktioniert auch so, funktioniert bei mir aber auch ohne conn.close(), kann sein das es an meinen Framework liegt, habt es aber mal trotzdem mit reingemacht